src/gui/tray/ActivityItemActions.qml

```diff
@@ -24,71 +22,71 @@ RowLayout {
     signal triggerAction(int actionIndex)
     signal showReplyField()
 
-    Repeater {
-        id: actionsRepeater
-        // a max of maxActionButtons will get dispayed as separate buttons
-        model: root.linksForActionButtons
+    model: root.linksForActionButtons
 
-        ActivityActionButton {
-            id: activityActionButton
+    Button {
```

Comment:
Why not use CustomButton? Or ActivityActionButton?

Reply:
I want to have a smaller button to have more compact layout

Reply:
(for reply button)

Reply:
Custom button can be very compact, see the "Sync now" button